src/components/pill/pill.stories.js

```diff
@@ -6,7 +6,6 @@ import { text, select, boolean } from '@storybook/addon-knobs';
 import { action } from '@storybook/addon-actions';
 import Pill from './pill.component';
 import OptionsHelper from '../../utils/helpers/options-helper';
-import { isClassic } from '../../utils/helpers/style-helper';
```

Comment:
I think it would be better to remove `import { THEMES } from '../../style/themes';`, keeping `isClassic` and then just passing the theme object to the helper function

Reply:
^ agree with Ed, I should have done it like that first time around, great catch!